.github/workflows/ci.yml

```diff
@@ -45,6 +45,44 @@ jobs:
         run: |
           bash scripts/format.sh
 
+  Benchmarks:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Ray Beam Runner
+        run: |
+          pip install -e .[test]
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
+          pip install -U "ray @ https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp38-cp38-manylinux2014_x86_64.whl"
+      - name: Run Benchmark
+        run: |
+          ray start --head
+          pytest -v benchmark/basic.py --benchmark-only --benchmark-json bm-result.json
+          cat bm-result.json
+          ray stop
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: Python Benchmark with pytest-benchmark
+          tool: 'pytest'
+          output-file-path: bm-result.json
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          auto-push: false
+          # Show alert with commit comment on detecting possible performance regression
+          alert-threshold: '200%'
+          comment-on-alert: true
+          fail-on-alert: true
+
```

Comment:
This runs on pull requests, right? Should we avoid storing the results since it's just a PR?

Reply:
let me take a look at this. this might be due to some issue that my code is based on old trunk.